halpybot/packages/announcer/announcer.py

```diff
@@ -186,6 +186,15 @@ async def announce(self, announcement: str, args: Dict, client: HalpyBOT):
         except Exception as announcement_exception:
             logger.exception("An announcement exception occurred!")
             raise AnnouncementError(Exception) from announcement_exception
```

Comment:
out of scope, but passing the base exception class to `AnnouncementError` is sus.